src/app/beer_garden/db/mongo/api.py

```diff
@@ -141,11 +141,12 @@ def check_connection(db_config: Box):
         # The 'connect' method won't actually fail
         # An exception won't be raised until we actually try to do something
         logger.debug("Attempting connection")
-        conn.server_info()
+        # conn.server_info()
```

Comment:
Two things:

* In general, leaving commented out lines of code isn't desirable.  We've got the git history, so if we need to resurrect something we'd do it that way.
* We need *something* to happen here.  The `conn.server_info()` call is an arbitrary one that's just a proxy for checking that we can connect to the database.  The easiest thing is probably to keep it and just break the try except into two separate blocks:

```python
    try:
        # Set timeouts here to a low value - we don't want to wait 30
        # seconds if there's no database
        conn = connect(
            alias="aliveness",
            db=db_config["name"],
            socketTimeoutMS=1000,
            serverSelectionTimeoutMS=1000,
            **db_config["connection"],
        )
    except ConnectionFailure as ex:
        logger.debug(ex)
        return False

    try:
        # The 'connect' method won't actually fail
        # An exception won't be raised until we actually try to do something
        logger.debug("Attempting connection")
        conn.server_info()
    except Exception as ex:
        logger.debug(ex)
        return False

    return True
```

The reason to do this is that you don't want to catch *all* exceptions for the `connect` call.  For instance, if someone changed the structure of `db_config` such that it no longer has a `name`, you don't want the `KeyError` that would get raised to be caught.

Catching all exceptions for the `conn.server_info()` call isn't ideal either, but by the time we get there all of the input has already been provided and it largely doesn't matter what the exception might be.